language/stdlib/modules/VASP.move

```diff
@@ -21,22 +22,23 @@ module VASP {
     /// A singleton resource allowing this module to publish limits definitions and accounting windows
     resource struct VASPOperationsResource { limits_cap: AccountLimitMutationCapability }
 
-    const ENOT_GENESIS: u64 = 0;
-    const ENOT_A_REGISTERED_CURRENCY: u64 = 1;
-    const EINVALID_SINGLETON_ADDRESS: u64 = 2;
-    const ENOT_LIBRA_ROOT: u64 = 3;
+    const EVASP_OPERATIONS_RESOURCE: u64 = 0;
+    const EPARENT_OR_CHILD_VASP: u64 = 1;
+    const ETOO_MANY_CHILDREN: u64 = 2;
+    const ENOT_A_VASP: u64 = 3;
     const ENOT_A_PARENT_VASP: u64 = 4;
-    const ENOT_A_VASP: u64 = 5;
-    const EALREADY_A_VASP: u64 = 7;
-    const ETOO_MANY_CHILDREN: u64 = 8;
+
 
     /// Maximum number of child accounts that can be created by a single ParentVASP
     const MAX_CHILD_ACCOUNTS: u64 = 256;
 
     public fun initialize(lr_account: &signer) {
-        assert(LibraTimestamp::is_genesis(), ENOT_GENESIS);
-        assert(Roles::has_libra_root_role(lr_account), ENOT_LIBRA_ROOT);
-        assert(Signer::address_of(lr_account) == CoreAddresses::LIBRA_ROOT_ADDRESS(), EINVALID_SINGLETON_ADDRESS);
+        LibraTimestamp::assert_genesis();
+        Roles::assert_libra_root(lr_account);
```

Comment:
Missing `CoreAddresses::assert_libra_root(lr_account);` here?

Reply:
Again, happens in Roles now.